packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -180,11 +181,11 @@ export class Table extends Construct {
    */
   public static grantListStreams(principal?: iam.IPrincipal): void {
```

Comment:
should the awslint rule also be applied to static methods?

Reply:
Probably. One problem with the statics is that we won't have a `scope` to attach a warning to.

Reply:
Probably not the end of the world